### DISCUSSION TITLE
Codespaces plugin is not updated for Jetbrains IDE‘s

### DISCUSSION THREAD

**Discussion by cwhde** [community/community#142326] [Codespaces]
### Select Topic Area

General

### Body

As seen on the extensions plugin at the [JetBrains marketplace](https://plugins.jetbrains.com/plugin/20060-github-codespaces), the current plugin version doesn‘t support errors on the latest releases for the IDE‘s, and in JetBrains Rider as well as GateWay there will be an error when trying to connect to a codespace. I haven‘t seen any information on this yet, and would like to ask if someone knows how long for the plugin to be updated?
Thanks

**Comment by moranenzo** [comment]
<p>Hi there!</p>

<p>Thanks for bringing this issue to light regarding the Codespaces plugin for JetBrains IDEs! It's definitely frustrating when tools don’t work as expected, especially with recent updates.</p>

<p>While I don’t have specific information on when the plugin will be updated, I recommend keeping an eye on the JetBrains Marketplace or the GitHub discussions for any announcements. You might also consider reaching out directly to JetBrains support for more insight!</p>

<p>Hope this helps, and feel free to share any further updates!</p>

**Comment by ebob9** [comment]
This appears to again be related to the [Github Codespaces Plugin ](https://plugins.jetbrains.com/plugin/20060-github-codespaces) not being updated. It needs updated to be compatible with the latest versions. It's a similar issue to #78982 - The plugin needs updated yet again.

@github-staff - Is there a resourcing/constraint that is preventing you from keeping the plugin up-to-date? I understand there is a larger focus on VS code, however I know of several JetBrains-focused companies that are investigating moving to other solutions due to the lack of updates on this (Google CloudWorkstations, AWS CodeCatalyst, CodeCanvas, etc..) 

If there is a resource constraint - would you consider open-sourcing the plugin so the community can help maintain it? (Not sure if that's an option.)

**Reply by SIMULATAN** [reply]
Your link - which, judging by a quick google search, appears to be correct - is now broken / 404s. [There's no search results either](https://plugins.jetbrains.com/search?excludeTags=internal&products=androidstudio&products=appcode&products=aqua&products=clion&products=dataspell&products=dbe&products=fleet&products=go&products=idea&products=idea_ce&products=mps&products=phpstorm&products=pycharm&products=pycharm_ce&products=rider&products=ruby&products=rust&products=webstorm&products=writerside&search=github+codespaces). It seems as if GitHub or JetBrains took the plugin down entirely. What a shame.

**Comment by heathdutton** [comment]
The plugin was removed from the JetBrains marketplace entirely (no longer a case of "not updated"). For those looking for an alternative, I put together a Toolbox plugin:

https://github.com/heathdutton/github-codespaces-jetbrains-toolbox

It shows your codespaces in Toolbox and lets you connect via SSH. Works with IntelliJ, PyCharm, WebStorm, and other IDEs.

Manual SSH also works: `gh codespace ssh -c <name>`
